Accessories/ExpansionBoard_uInverter/REV20210506A/README.md

```diff
@@ -0,0 +1,30 @@
+# uInverter Board
+
+The uInverter (micro-inverter) board is an accessory for the AMDC. The purpose of this board is to serve as a low-cost, micro-level prototype board to demonstrate three-phase current regulation using the AMDC.
+
+
+This folder contains the compiled design outputs for the uInverter expansion board. This includes schematics, images of the PCB layout, the bill of materials (BOM), and the files needed to order fully populated boards. 
+
+## PCB Properties
+| Question          | Answer        |
+|-------------------|---------------|
+| Size              | 3.6" x 6.3"   |
+| Number of Layers  | 2             |
+| Number of components | 66         |
+|Number of holes    | 205           |
+| Thickness         | 1.6mm         |
+| Min Track/Spacing | 6/6mil          |
+| Min Hole Size     | 0.3mm           |
+| Silkscreen        | Top Only      |
+| Part Count        | 66            |
+
+
+## Connectors and connections
+
+Please check the [docs folder](../docs/) for more details on the connectors and pin-outs.
+
+Documentation about the design of uInverter board is located in [docs folder.](../docs/)
```

Comment:
Odd to see this same folder linked twice in two subsequent sections that are each their own paragraph.

Reply:
Updated.